packages/manager/modules/pci/src/projects/project/additional-ips/order/order.controller.js

```diff
@@ -197,12 +256,31 @@ export default class AdditionalIpController {
       .getGateways(this.projectId, this.ip.region.name)
       .then((gateways) => {
         this.gateways = gateways;
+        [this.gateway] = gateways; // TODO which gateway to take? waiting answer from PM
+        if (gateways.length === 0) {
+          this.getGatewayPlan().then((plan) => {
+            this.gatewayPrice = `${plan.pricings[0].price +
+              plan.pricings[0].tax}${this.currencySymbol}`;
+          });
+        }
       })
       .finally(() => {
         this.loadingGatewayDetails = false;
       });
   }
 
+  getGatewayPlan() {
+    return this.publicCloudCatalog.addons.find(
+      (addon) => addon.product === 'publiccloud-gateway-s',
+    );
+  }
+
+  getFailoverIpPlan() {
+    return this.publicCloudCatalog.addons.find(
+      (addon) => addon.product === 'publiccloud-floatingip-floatingip',
```

Comment:
same here, are we always going to have only one plan `publiccloud-floatingip-floatingip`?

Reply:
yes there will be only one.